src/angularjs/src/app/components/filters/analysis-job-filter.html

```diff
@@ -2,23 +2,29 @@
     <th>ID</th>
     <th>
         <div class="dropdown">
-        <input name="neighborhood-filter" ng-model="ctl.statusFilter" title="Filter by Status"
-            placeholder="Status" type="text" class="form-control"
-            uib-typeahead="status for status in ctl.statuses |
-                           filter: $viewValue |
-                           limitTo: 8">
+        <input name="neighborhood-filter"
+               title="Filter by Status"
+               placeholder="Status"
+               ng-model="ctl.statusFilter"
+               ng-model-options="{ debounce: 400 }"
```

Comment:
Apparently we need to debounce both via `ng-model-options` and `typeahead-wait-ms` for the debounce to work as expected. Only setting one or the other leads to the debounce not taking effect.